assets/css/globals.css

```diff
@@ -22,6 +22,18 @@ button {
   }
 }
 
+input {
```

Comment:
@by-tim Is there a more Tailwind-y way to accomplish this? We should avoid having CSS like this if possible.